acquisitions/src/main/resources/thunderjet/mod-orders/features/check-new-tags-in-central-tag-repository.feature

```diff
@@ -33,7 +33,7 @@ Feature: Check new tags created in central tag repository
     {
       "tagList": [
         "check - central - tag",
-        " Another  _Tag"
+        " Another  -Tag"
```

Comment:
Is it known reason why it is working with "-" but not working with "_" now?

Reply:
It's separate issue connected with upgrade to java 17/spring boot 3, new ticket has been created for the Spitfire team: https://issues.folio.org/browse/MODTAG-105 
For now we agreed that we can avoid this "_" symbol in our mod-orders karate test.
Spitfire will create separate karate test to test "_" after issue will be resolved